roles/ceph-osd/tasks/build_devices.yml

```diff
@@ -35,3 +35,36 @@
   when:
     - osd_scenario == 'non-collocated'
     - not osd_auto_discovery
+
+- name: get OSD journal partitions
+  shell: blkid | awk '/ceph journal"/ { sub (":", "", $1); print $1 }'
```

Comment:
please use the `command` module with the following command: `blkid -o device -t PARTLABEL="ceph journal"`